home/tests/required_fields.csv

```diff
@@ -0,0 +1,3 @@
+structure,message,page_id,slug,parent,web_title,locale

```

Comment:
Are structure and page_id required for the import? What happens if we remove those?

Reply:
The import on the UI still works s we can remove those as well